zio/parquetio/reader.go

```diff
@@ -125,6 +125,11 @@ func simpleParquetTypeToZngType(typ HandledType) zng.Type {
 	panic(fmt.Sprintf("unhandled type %d", typ))
 }
 
+type ReaderOpts struct {
+	columns                []string
+	ignoreUnhandledColumns bool
```

Comment:
We should export these so they're available outside this package.

Reply:
whoops indeed we should.